src/master-playlist-controller.js

```diff
@@ -614,6 +614,8 @@ export class MasterPlaylistController extends videojs.EventTarget {
         this.requestOptions_.timeout = requestTimeout;
       }
 
+      this.masterPlaylistLoader_.load();
```

Comment:
Shouldn't this happen in the dash/hls playlist loader itself?

Reply:
You mean like add a `'mediachange'` listener to the constructor? I think we could. Does the same go for the `mainSegmentLoader_` method calls below this? Should they be moved as well?

Reply:
So the master playlist loader fires `mediachange`. It should probably start to load by itself after that too? Could totally be wrong, as I haven't looked at the code in a bit now. I think having the `mainSegmentLoader_` call `load` here is fine.